content/getting-started/directory-structure.md

```diff
@@ -121,6 +121,10 @@ publish = true
 # whether mark this article as a featured article. 
 # the featured articles will be shown on the home page
 featured = true
+# This can be set if you are the original author of an article, and have submitted a reformatted
+# version of your article for a zine site. This ensure that search engines will rank your original
+# and not impact SEO negatively.
+canonical = "URL of the orginal Article"
```

Comment:
I suggest to comment like this:

When articles are reposted from the original source, set a `canonical` property to help prevent duplicate content issues for search engines.

> For more information, please refer to the [Canonical link element
> ](https://en.wikipedia.org/wiki/Canonical_link_element) on Wikipedia.